submissions/ayokofemi/database/init.sql

```diff
@@ -0,0 +1,49 @@
+--for the foodTable--
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(20) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the vendorFood--
+CREATE TABLE vendor_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(250) NOT NULL,
+amount FLOAT(10,2) NOT NULL,
+PRIMARY KEY (id),
+FOREIGN KEY (food_id) REFERENCES food (id),
+FOREIGN KEY (admin_id) REFERENCES admin (id)
+);
+
+--for the adminsTable--
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250) NOT NULL,
+type  ENUM("bank_transfer","Card","payment_on_arrival","dollar") NOT NULL,
```

Comment:
This should not be here, instead you should have:  type  ENUM ('0', '1') NOT NULL DEFAULT '0'
The admins table will contain both the system admins of the food bag platform and the vendors. They both will be differentiated by type.The system admins can have a type of 0 and vendors a type of 1 when they are saved for differentiation.